src/main/java/ch/sbb/polarion/extension/pdf_exporter/widgets/BulkPdfExportWidgetRenderer.java

```diff
@@ -83,7 +98,7 @@ protected void render(@NotNull HtmlFragmentBuilder builder) {
 
             HtmlTagBuilder header = wrap.append().tag().div();
             header.attributes().className("header");
-            header.attributes().byName("document-type", getItemsType(itemsPrototype).name());
+            header.attributes().byName("document-type", PrototypeEnum.BaselineCollection.equals(itemsPrototype) ? PrototypeEnum.BaselineCollection.name() : getItemsType(itemsPrototype).name());
```

Comment:
This modification looks weird taking into account that there's `BaselineCollection` case in `getItemsType` method

Reply:
done